src/main/java/app/linguistai/bmvp/model/enums/FriendshipStatus.java

```diff
@@ -0,0 +1,16 @@
+package app.linguistai.bmvp.model.enums;
+
+public enum FriendshipStatus {
+    PENDING(0),
+    ACCEPTED(1);
+
+    private int value;
```

Comment:
Java enums already store numbers by default, it could be better if you just use something like the following `public enum FriendshipStatus {PENDING, ACCEPTED}`

Reply:
It can also stay like this if you wish, up to you